kpatch-build/create-diff-object.c

```diff
@@ -1578,6 +1578,10 @@ static void kpatch_replace_sections_syms(struct kpatch_elf *kelf)
 			    !strcmp(rela->sym->name, ".text..refcount"))
 				continue;
 
+			/* Skip powerpc cpu feature fixup relocations */
+			if (!strncmp(rela->sym->name, "__ftr_alt_", 10))
+				continue;
```

Comment:
Can we lump this check in with the above "don't have symbols associated with them" checks?  Then I think the comment isn't really needed, it's similar to the other checks in that the section doesn't have symbols associated with it.  The full backstory is available in the commit log for those who are interested.